### PR TITLE
fix USB starting when no applet is loaded, use UID as USB iSerial value

### PR DESCRIPTION
**Pull request by abarisani**

### CODE REVIEW

File: trusted_os/ctl.go [hunk 1]
```diff
@@ -115,6 +116,14 @@ func (ctl *controlInterface) Start() {
 		return
 	}
 
+	if !irq {
+		Control.Init()
+		Control.DeviceMode()
+		Control.Reset()
+
+		Control.Start(device)
```
Comment (AlCutter) [inline trusted_os/ctl.go:124]: Does `Start()` not return, or is it ok that we do this `Init`/`Mode`/`Reset`/`Start` immediately followed by further calls to `Init`/`DeviceMode`/`EnableInterrupts`?

Comment (abarisani) [reply trusted_os/ctl.go:124]: Start never returns.

Comment (AlCutter) [reply trusted_os/ctl.go:124]: Might be worth a comment to that effect (there's precedent for that elsewhere :)?

Comment (abarisani) [reply trusted_os/ctl.go]: Should be better in https://github.com/transparency-dev/armored-witness-os/pull/4/commits/5e8f038c18e10c19283617f063c8bbfcfa935a8d

**Pull request merged**